dashboard/app/models/pd/fit_weekend1819_registration.rb

```diff
@@ -0,0 +1,104 @@
+# == Schema Information
+#
+# Table name: pd_fit_weekend1819_registrations
+#
+#  id                :integer          not null, primary key
+#  pd_application_id :integer
+#  form_data         :text(65535)
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#
+# Indexes
+#
+#  index_pd_fit_weekend1819_registrations_on_pd_application_id  (pd_application_id)
+#
+
+class Pd::FitWeekend1819Registration < ActiveRecord::Base
+  include Pd::Form
+
+  belongs_to :pd_application, class_name: 'Pd::Application::ApplicationBase'
+
+  YES = 'Yes'.freeze
+  NO = 'No'.freeze
+  YES_OR_NO = [YES, NO].freeze
+
+  def self.options
+    {
+      ableToAttend: YES_OR_NO,
+      dietaryNeeds: [
+        'None',
+        'Vegetarian',
+        'Vegan',
+        'Halal',
+        'Gluten Free',
+        'Food Allergy (please list):',
+      ],
+      liveFarAway: YES_OR_NO,
+      addressState: get_all_states_with_dc.to_h.values,
+      howTraveling: [
+        'I will drive by myself',
+        'I will carpool with another FiT Weekend attendee',
+        'Flying',
+        'Amtrak or regional train service',
+        'Public transit (e.g., city bus or light rail)',
+      ],
+      needHotel: YES_OR_NO,
+      needAda: [YES, NO, "Other (please explain):"],
```

Comment:
Do we need the "other" options here as well as at https://github.com/code-dot-org/code-dot-org/pull/20218/files#diff-c67abb1ceb40fdf645dedcabe2fc5586R133 ?

Reply:
We do; the other option here is for rendering them as a radio button option, the `radioButtonsWithAdditionalTextFieldsFor` method specifies that an extra text field should also be attached to _that specific radio button option_.

Best practice would be to put the common string in some shared constants [like we do for the teachercon registration form](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/shared_constants/pd/teachercon1819_registration_constants.rb) but I went for the lazy option